src/global.css

```
@@ -1,17 +1,25 @@
-@import url('https://fonts.googleapis.com/css2?family=Quicksand&display=swap');
+@import url("https://fonts.googleapis.com/css2?family=Quicksand&display=swap");
 
 *,
 *::after,
 *::before {
   box-sizing: border-box;
 }
 
+:root {
+  --color-primary: #91c788;
+  --color-secondary: #ff8473;
+  --color-maintext: #000000;
+  --color-subcontent: #a9a9a9;
+  --font-family: "Signika", sans-serif;
+}
+
```

```suggestion
:root {
  --color-primary: #91c788;
  --color-secondary: #ff8473;
  --color-text-main: #000000;
  --color-content-sub: #a9a9a9;
  --color-background-main: #E5E5E5;
  --font-family: "Signika", sans-serif;
}

```

The first two is just my preference how I would write it, although I would add the main background color, since it's one of the most important colors if you want to change to dark mode for example. Otherwise, looks good to me  🥇